docs/en/core-concepts.md

```diff
@@ -0,0 +1,10 @@
+# Core concepts
+
+We will be learning the core concepts in Vuex in this chapters. They are
+- State
+- Getters
+- Mutations
+- Actions
+- Modules
```

Comment:
It would be great if these items have the link to the corresponding chapter :slightly_smiling_face: